packages/table/test/selectableTests.tsx

```diff
@@ -31,13 +31,15 @@ describe("DragSelectable", () => {
 
     it("does click selection", () => {
         const onSelection = sinon.spy();
+        const onFocus = sinon.spy();
```

Comment:
Also it's a lot more straightforward to always pass down a callback and filter at the top than it is to conditionally pass down a callback and call it depending on existence at the bottom.